db/migrate/20240718195301_update_form_answers_table.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateFormAnswersTable < ActiveRecord::Migration[7.1]
```

Comment:
You should be able to combine AddOrganizationToFormAnswers and UpdateFormAnswersTable migrations. You created both of these on your branch, right? So you should be able to roll back the last two migrations and merge them together (you can keep the UpdateFormAnswersTable class name) and then run migrations again.

Reply:
Yeah, there is no need to split them up. They both update the FormAnswer table. I combined them and pushed the changes! 

I ended up using `rails db:migrate:down VERSION=20240716194544` (AddOrganizationToFormAnswers) to undo the changes of that migration and then added the Organization reference to the other.

I had some trouble using rollback. Something about NotNullViolation: ERROR. However, I think I could have done something similar to what you've done in some of your migration files with `def up` and `def down`. I haven't gone too deep into migrations and it's something I need more work on!

`def up` would be the changes made? and `def down` would be how to reverse those changes? So a command like rails rollback could work? I accomplished the same thing but in a roundabout way, I suppose. I need to look into that more!

Reply:
Ah yes I think you encountered that because you removed a null constraint, so undoing that would cause an error given there are records with null? That's just a guess. 

There's definitely a few ways to write migrations. We are soon adding the Strong Migrations gem that will ensure some steps are made in these files.